todo_list/models.py

```diff
@@ -1,3 +1,11 @@
 from django.db import models
 
-# Create your models here.
+
+class Page(models.Model):
+    title = models.CharField(max_length=10)
```

Comment:
@quratulain25 Add **create_at** and **modified_at** date time fields for every model where entries can be changed any time.